pkgs/build-support/build-fhs-userenv/chrootenv.c

```diff
@@ -161,7 +230,7 @@ int main(int argc, char *argv[]) {
 
     argv++;
 
-    if (execvpe(*argv, argv, env_build(env_whitelist, LEN(env_whitelist))) < 0)
+    if (execvpe(*argv, argv, env_filter(environ)) < 0)
```

Comment:
I think the code for the blacklist approach would be much simpler if you just used `execvp()` and just modified the current environment with `unsetenv()`.

Reply:
I agree, I'm going to do that. Thanks!